README.md

```diff
@@ -159,6 +159,13 @@ prefix or, if set to an empty string, completely disable prefixing:
 const parser = new N3.Parser({ blankNodePrefix: '' });
 ```
 
+By default, `N3.Parser` does not use the `log:isImpliedBy` predicate when parsing
```

Comment:
Maybe describe the default behavior?
Something like:

The parser can output a backwards chaining rule such as `_:q <= _:p.` in two ways:
- as `_:p log:implies _:q.` (default)
- as `_:q log:isImpliedBy _:p.` (when the `isImpliedBy` flag is  set to `true`)